src/modules/InternetDevices.rb

```diff
@@ -1,116 +0,0 @@
-# encoding: utf-8
```

Comment:
But what about this use? https://github.com/yast/yast-live-installer/blob/4a62460f1a8abaea00be50e671c719711c430a57/src/clients/inst_live_prepare_net_test.rb#L56